source/billing.txt

```diff
@@ -120,35 +121,61 @@ There are several types of requests:
   For more information on sync operations, see :ref:`Atlas Device Sync Protocol
   <sync-protocol>`.
 
-All incoming requests count toward your monthly usage except for the
+All incoming requests count toward your daily usage except for the
 following:
 
 - Requests to an authentication provider
 - Requests blocked by an application's :ref:`IP Access List <manage-ip-access>`
 - Requests to an invalid HTTP endpoint
 
-**Price:** $2.00 / 1,000,000 requests ($0.000002 / request)
+.. _request-tier-pricing:
 
-**Formula:** (Function Executions + Trigger Executions + GraphQL/Webhook/HTTPS Endpoint Requests + Sync Updates) * $0.000002
+**Price:** 
 
-**Free Tier Threshold:** 1,000,000 :ref:`requests <billing-requests>` or 500
-hours of :ref:`compute <billing-compute>` or 10,000 hours of :ref:`sync
-<billing-sync>` runtime (whichever occurs first)
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 25 25
+
+   * - Billing Tier
+     - Request Volume (Requests/Day)
+     - Price (per Million Requests)
+     - Price (per Request)
+
+   * - Tier 1
+     - Less than 25 Million
+     - $2.00 USD
+     - $0.000002 USD
+
+   * - Tier 2
+     - 25 Million - 250 Million
+     - $1.00 USD
+     - $0.000001 USD
+
+   * - Tier 3
+     - Greater than 250 Million
+     - $0.75 USD
+     - $0.00000075 USD  
+
+**Formula:** (Function Executions + Trigger Executions + GraphQL/Webhook/HTTPS Endpoint Requests + Sync Updates) * (Price per Request Depending on Tier)
+
+**Free Tier Threshold:** 50,000 :ref:`requests <billing-requests>` or 25 hours of
+  :ref:`compute <billing-compute>` or 30,000 minutes of :ref:`sync
+  <billing-sync>` runtime (whichever occurs first)
```

Comment:
Technical Reviewer: If you pass one of the free tier limits, for example the 0.5 GB data transfer, do the others (sync minutes, requests, compute hours) get charged outside of the free tier, even if they are under their respective free tier limit?

Reply:
Each limit is independent, so they get Requests AND Data AND Sync Minutes AND Compute each day. Instead of the OR it was previously in the formula. So to your question - no they would not be charged for the limits that are under.